src/file/file.service.ts

```diff
@@ -16,85 +14,6 @@ export const FolderContentType = 'application/vnd.drive.folder';
  * olderID is an objectID of an existing file of type folder.
  */
 export class FileService {
```

Comment:
So why does this file exists?

Reply:
this contains the logical part of the methods in the file service